app/views/collections/_show_document_list_row.html.erb

```diff
@@ -1,12 +1,10 @@
 <% id = document.id %>
+<% presenter = document.is_a?(GenericWork) ? Sufia::WorkShowPresenter.new(document, nil) : nil %>
```

Comment:
Is there already a presenter defined here that is the presenter of the collection?